src/meta/test/BalanceIntegrationTest.cpp

```diff
@@ -93,15 +87,15 @@ TEST(BalanceIntegrationTest, DISABLED_LeaderBalanceTest) {
         serverContexts.emplace_back(std::move(sc));
     }
 
-    ret = mClient->createSpace("storage", partition, replica).get();
+    auto ret = mClient->createSpace("storage", partition, replica).get();
     ASSERT_TRUE(ret.ok());
-    sleep(FLAGS_load_data_interval_secs + 1);
-    sleep(FLAGS_raft_heartbeat_interval_secs);
+    sleep(FLAGS_load_data_interval_secs + FLAGS_raft_heartbeat_interval_secs + 3);
```

Comment:
FLAGS_load_data_interval_secs + FLAGS_raft_heartbeat_interval_secs + 1 ?

Reply:
+3 is due to the UT occasionally failed.